schemas/ajv.absolutePath.js

```diff
@@ -1,29 +1,33 @@
 "use strict";
 
 const getErrorFor = (shouldBeAbsolute, data, schema) => {
-	const message = shouldBeAbsolute ?
-		`The provided value ${JSON.stringify(data)} is not an absolute path!`
-		: `A relative path is expected. However the provided value ${JSON.stringify(data)} is an absolute path!`;
+	const message = shouldBeAbsolute
+		? `The provided value ${JSON.stringify(data)} is not an absolute path!`
+		: `A relative path is expected. However the provided value ${JSON.stringify(
+				data
+			)} is an absolute path!`;
 
 	return {
 		keyword: "absolutePath",
 		params: { absolutePath: data },
 		message: message,
-		parentSchema: schema,
+		parentSchema: schema
 	};
 };
-module.exports = (ajv) => ajv.addKeyword("absolutePath", {
-	errors: true,
-	type: "string",
-	compile(expected, schema) {
-		function callback(data) {
-			const passes = expected === /^(?:[A-Za-z]:\\|\/)/.test(data);
-			if(!passes) {
-				callback.errors = [getErrorFor(expected, data, schema)];
+
+module.exports = ajv =>
```

Comment:
Why code format changed?

Reply:
@montogeek `prettier`

Reply:
I understand that src code was already formatted with prettier, he didn't changed anything to increment line width

Reply:
This file never was reformatted by prettier.

Reply:
looks like the `schemas` folder is not included in the prettier scripts.

Reply:
Yes, the file was never formatted, when I first opened it there were 19 ESLint violations.